src/grdeviceside.cpp

```diff
@@ -468,8 +468,11 @@ namespace rhost {
             ///////////////////////////////////////////////////////////////////////
 
             std::unique_ptr<ide_device> ide_device::create(const boost::uuids::uuid& device_id, std::string device_type, double width, double height, double resolution) {
+#ifdef _WIN32
                 pDevDesc dd = static_cast<pDevDesc>(rhost::msvcrt::calloc(1, sizeof(DevDesc)));
-
+#else
+                pDevDesc dd = static_cast<pDevDesc>(calloc(1, sizeof(DevDesc)));
```

Comment:
Since this is used several times, and the two versions are really the same function (just from different DLLs), can you make a wrapper that contains the ifdef inside? Something like `R_calloc`.